### PR TITLE
Enables Travis CI and Slack notification

### PR DESCRIPTION
**Pull request by jwgwalton**

### CODE REVIEW

**Review by emilioward — APPROVED** [review]

**Review by benjefferies — APPROVED** [review]

**Comment by jwgwalton** [comment]
Don't merge until Travis turned on

**Comment by benjefferies** [comment]
Lets check travis on this one before merging?

**Comment by madeye-matt** [comment]
LOL just going through merging all PRs with 2 approves - didn't see the comment so merged it 🙂

**Pull request merged**
